include/receiver_cadmium.hpp

```diff
@@ -108,7 +108,7 @@ class Receiver {
         }
 
         friend std::ostringstream& operator<<(std::ostringstream& os, 
-		            const typename Receiver<TIME>::state_type& i) {
+            const typename Receiver<TIME>::state_type& i) {
             os << "ack_num: " << i.ack_num; 
```

Comment:
I would suggest to not give any space before and after <<. i.e., os<<"ack_num: "<<i.ack_num;

Reply:
I think spaces are OK around << character

Reply:
When we checked with prof, she said spaces are not required. But to be on the same page, we can double check with prof today during lecture.

Reply:
I am sure spaces around << are OK. I saw examples online.

Reply:
okay, works for me.